src/code/z_play.c

```diff
@@ -11,8 +12,8 @@ TransitionTile sTransitionTile;
 s32 gTransitionTileState;
 VisMono sPlayVisMono;
 Color_RGBA8_u32 gVisMonoColor;
-Struct_80140E80 D_801F6D38;
-Struct_80140E80* D_801F6D4C;
+VisFbuf sPlayVisFBuf;
+VisFbuf* sPlayVisFBufInstance;
```

Comment:
My bad. I just noticed everywhere else uses `VisFbuf` (lowercase b). Would you mind changing these to match that?